primitives/tf_encrypted/primitives/sodium/python/easy_box_test.py

```diff
@@ -22,24 +22,24 @@ def test_seal(self):
         _, sk_s = easy_box.gen_keypair()
         pk_r, _ = easy_box.gen_keypair()
 
-        plaintext = tf.constant([1, 2, 3, 4], shape=(2, 2), dtype=tf.uint8)
+        plaintext = tf.constant([1, 2, 3, 4], shape=(2, 2), dtype=tf.float32)
```

Comment:
perhaps copy the tests instead so we continue to support both `tf.uint8` and `tf.float32`